File: ecbs-os/frontend/src/lib/deploymentCompletionData.ts
import type { AnalysisSummaryRow } from "@/lib/analysisData";

export type DeploymentCompletionData = {
  clientName: string;
  closureCards: AnalysisSummaryRow[];
  deploymentId: string;
  documentRows: AnalysisSummaryRow[];
  equipmentRows: AnalysisSummaryRow[];
  identityRows: AnalysisSummaryRow[];
  message: string;
  projectName: string;
  siteName: string;
  state: string;
  status: string;
  updatedAt: string;
};

File: ecbs-os/backend/src/ECBS.Application/ScreenData/DeploymentCompletionData.cs
namespace ECBS.Application.ScreenData;

public sealed record DeploymentCompletionData(
    IReadOnlyList<AnalysisSummaryRow> ClosureCards,
    IReadOnlyList<AnalysisSummaryRow> DocumentRows,
    IReadOnlyList<AnalysisSummaryRow> EquipmentRows,
    IReadOnlyList<AnalysisSummaryRow> IdentityRows,
    string ClientName,
    string DeploymentId,
    string Message,
    string ProjectName,
    string SiteName,
    string State,
    string Status,
    string UpdatedAt);

